che-theia-plugins.yaml

```diff
@@ -195,14 +195,14 @@ plugins:
         - redhat/vscode-yaml
   - repository:
       url: 'https://github.com/camel-tooling/vscode-camelk'
-      revision: 0.0.23
+      revision: 0.0.24
```

Comment:
Doesn't look like this revision exists yet?

Reply:
omg, I forgot to create the tag...

Creating it

Reply:
done